### PR TITLE
Upgrade: php to 7.0.14, composer 1.2.4, drush 8.1.8

### PR DESCRIPTION
**Pull request by andypost**
Clean-up useless env
Fix user creation for php:7-fpm (still using root cos needed for projects)
Upgrade: 
- php to 7.0.14 & 5.6.29
- composer 1.2.4
- drush 8.1.8
- php*-apcu & xdebug now moved out of testing

PS: images already pushed

### CODE REVIEW

**Comment by andypost** [comment]
Now waiting for https://github.com/alpinelinux/aports/pull/601

**Comment by andypost** [comment]
Last PR accepted https://github.com/alpinelinux/aports/pull/598

File: php56-fpm/php.ini [hunk 1]
```diff
@@ -0,0 +1,38 @@
+; PHP limits
+memory_limit = 1024M
```
Comment (andriyun) [inline php56-fpm/php.ini]: Not clear why we need so big memory limit

File: php7-fpm/php.ini [hunk 2]
```diff
@@ -1,16 +1,19 @@
 ; PHP limits
-memory_limit = 512M
+memory_limit = 1024M
```
Comment (andriyun) [inline php7-fpm/php.ini]: same here

File: php56-fpm/php.ini (hunk 1 above)
Comment (jbguerraz) [inline php56-fpm/php.ini]: +1

File: php7-fpm/php.ini (hunk 2 above)
Comment (jbguerraz) [inline php7-fpm/php.ini]: +1

File: php56-fpm/php.ini (hunk 1 above)
Comment (andypost) [inline php56-fpm/php.ini]: reverted back to 512M as fpm using inside its conf file

**Comment by andypost** [comment]
synced all files to use 512M

File: docker-compose.yml [hunk 3]
```diff
@@ -17,6 +17,7 @@ services:
       - mailhog
     restart: always
     command: php -t /srv -S 0.0.0.0:80 .ht.router.php
+#    command: php -d zend_extension=xdebug.so -t /srv -S 0.0.0.0:80 .ht.router.php
```
Comment (andriyun) [inline docker-compose.yml]: @andypost @jbguerraz  Lets move docker-compose.yml, *.sh and *.make.yml and all environments related  files to separate repo. And put references to proper images from docker hub.

In this case we will have more clear and simpler support process

Comment (andypost) [inline docker-compose.yml]: I guess it should go to https://github.com/skilld-labs/skilld-docker-container

Comment (andriyun) [inline docker-compose.yml]: Anyway I don't see any reason to store environment related files in this repo.

Comment (andypost) [inline docker-compose.yml]: This line is needed because we bundle XDebug now, so here should be a quick way to use it

Comment (andypost) [inline docker-compose.yml]: See https://github.com/alpinelinux/aports/blob/master/community/php-xdebug/APKBUILD#L58

**Pull request merged**